http_requests.py

```diff
@@ -163,13 +168,16 @@ def _request(self,
             if 'content-type' not in headers:
                 headers['content-type'] = 'application/x-yaml'
 
-        result = method(
-            url,
-            headers=headers,
-            auth=self.auth,
-            verify=self.verify_ssl,
-            **kwargs
-        )
+        try:
+            result = method(
+                url,
+                headers=headers,
+                auth=self.auth,
+                verify=self.verify_ssl,
+                **kwargs
+            )
+        except Exception:
+            _log_stacktrace_to_els(url=url)
```

Comment:
- pass stacktrace explicitly (mv tracback-call here)
- instead of doing early-exit in called function, check whether running on cicd here, and simply do not call the function in this case
- pass all args (not just URL)
- explicitly capture the exception (`as e` ...)
- re-raise _here_